static/app/components/events/eventReplay/replayContent.tsx

```diff
@@ -1,91 +1,134 @@
+import {useMemo} from 'react';
 import styled from '@emotion/styled';
 
+import Button from 'sentry/components/button';
 import DateTime from 'sentry/components/dateTime';
 import Duration from 'sentry/components/duration';
+import EventDataSection from 'sentry/components/events/eventDataSection';
 import Placeholder from 'sentry/components/placeholder';
 import {Provider as ReplayContextProvider} from 'sentry/components/replays/replayContext';
 import ReplayView from 'sentry/components/replays/replayView';
+import {relativeTimeInMs} from 'sentry/components/replays/utils';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
+import {Event} from 'sentry/types/event';
 import useFullscreen from 'sentry/utils/replays/hooks/useFullscreen';
 import useReplayData from 'sentry/utils/replays/hooks/useReplayData';
 import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 
 type Props = {
+  event: Event;
   orgSlug: string;
   replaySlug: string;
 };
 
-function ReplayContent({orgSlug, replaySlug}: Props) {
+function ReplayContent({orgSlug, replaySlug, event}: Props) {
   const {fetching, replay, fetchError} = useReplayData({
     orgSlug,
     replaySlug,
   });
   const {ref: fullscreenRef, toggle: toggleFullscreen} = useFullscreen();
+  const eventTimestamp = event.dateCreated;
```

Comment:
This isn't the right timestamp to use. It should be `startedAt` from the replay.